DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateGroupsController.cs

```diff
@@ -47,6 +57,43 @@ public IActionResult GroupDelegates(int groupId, int page = 1)
             return View(model);
         }
 
+        [HttpGet]

```

Comment:
Could we add some tests for the controller too? e.g. don't remove if not confirmed, wrong group id & delegate id combo etc?